db/migrations/2021_12_29_000000_users_settings_add_email_goody.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Engelsystem\Migrations;
+
+use Engelsystem\Database\Migration\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class UsersSettingsAddEmailGoody extends Migration
+{
+    use Reference;
+
+    /**
+     * Run the migration
+     */
+    public function up()
+    {
+        $this->schema->table(
+            'users_settings',
+            function (Blueprint $table) {
+                $table->boolean('email_goody')->default(false)->after('email_human');
```

Comment:
Migrations work on DB level. Booleans are stored as numbers: `false` should be `0`.
May work by accident because it is going to be casted somewhere :shrug: 

Typed default values should be set in the models
https://laravel.com/docs/8.x/eloquent#default-attribute-values

Reply:
It definitely gets converted totally fine and thus is imho more clear here?

And yeah, that should be eventually done in all models at some point (defaults but also explicit casting definitions)

Reply:
> And yeah, that should be eventually done in all models at some point (defaults but also explicit casting definitions)

Jep. Because default values only defined per DB will only apply after saving the model.

Reply:
Its now also explicitly converted in the model :tada: